helm-chart-sources/appscope/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: appscope
+version: 1.0.0
```

Comment:
The other charts sync the `version` and `appVersion` fields, except for pre-releases because of SemVer.